libs/sdk-ui-pivot/src/impl/tests/stickyRowHandler.test.ts

```diff
@@ -141,8 +146,9 @@ describe("stickyRowHandler", () => {
         });
 
         describe("current scroll top belongs to different line than the last one", () => {
-            const fakeGetDisplayedRowAtIndex = (): any => ({ data: { a_123: "123" } });
-            const fakeGridApi = getFakeGridApi(fakeGetDisplayedRowAtIndex);
+            const fakeGetDisplayedRowAtIndex = (): any => ({ data: { r_0: "123" } });
```

Comment:
This `r_0` etc. is leaking the logic of the prefixes from the factories. I wonder if we should have functions similar to `idRef` like `rowId(index: number)` and use it everywhere.

Reply:
some of us are not ok with using the same "factories" for both generating the content and expected data in tests

Reply:
Ok, fair enough